docs/README.md

```diff
@@ -4,13 +4,167 @@
+
+After installing these, open the codebase by File->Open Projects from FileSystem.
+Navigate to the magic-wifi/magicwifi folder. After selecting it, select finish to open the project.
+
+In the project there are folders for each major component and some necessary base folders.
+
+|   Folder      |  Info        |
+|---------------|--------------|
+| incldes | Necessary header and include folders for library functions |
+|  ATWINC1500_MSP   | All necessary driver information for the ATWINC WiFi module |
+| Debug | When a debug build is made the Debug folder is filled with necessary binaries and items |
+| driverlib | Contains main libraries for divers for the MSP430 included by the includes folders |
+| EtherCard_MSP430FR2355 | A custom ported version of the Arduino EtherCard Library, as only minimal functionality is needed and space on the MSP430 ROM chip is limited, only necessary parts of the library were ported |
+| FrameTranslator | A class called FrameTranslator, that translates Ethernet frames into LoRa packets adding parity and consistency |
+| LoRaRF_2.1.1_driver | Contains headers and classes to run the LoRa RF modules more smoothly in the main code driver |
+| targetConfigs | A base level configuration for CCStudio to build for the MSP430FR2355 |
+| defines.h | A number of const items that are used throughout the project. |
+| lnk_msp430fr2355.cmd | A linker file produced by CCStudio, no need to touch this file |
+| main.cpp | The main project and running codebase that builds from all other parts |
+
+To build and load the project, go to Project-> Build Project, proceed past the efficiency warnings. Afterward you can plug in your TI Dev Board and select Run-> Load.
+On the first load you must Select program to load, choose Browse project and navigate to debug-> magicwifi.out
+
+This should flash your firmware correctly.
+
+## Developing on PCB
+
+### NOTE: PCB is under redevelopment on KiCAD from Eagle and is therefore not stable to be produced.
+
+![image not found](images/Top_PCB_V1_2.png)
+Fig: An image of Version 1.2 of the MagicWiFi PCB, the first working version with modifications.
+
+If you a developing on a PCB version of MagicWiFi, to Flash binaries built for MSP430 you will need the [MSP430 Flasher](https://www.ti.com/tool/MSP430-FLASHER), and a TI JTAG flashing device to flash firmware to the MSP430.
+
+All necessary Components are listed in [docs/BillsOfMaterials/PCB-BOM](docs/BillsOfMaterials/PCB-BOM.md)
+
+### Major Issues
+
+One major issue is At time of writing Jan 10, 2024. The ATWINC1500 modules for sale on DigiKey have an earlier version of firmware than is necessary. On the module there is not a method of flashing new firmware. This was overcome in the development process by desoldering the ATWINC1500 from the module and soldering on an ATWINC1500 from a dev board that could have the firmware updated. A new method must be decided upon as this requires more specialized tools and is no user friendly.
+

```

Comment:
I know why it was there in the report, but I don't think this belongs here, right?
It seems like a worse version than someone looking at the actual header files, and seems almost guaranteed to immediately get out of sync with the originals :)

Reply:
fair. I have to remember general audience on the system